api/models/User.js

```diff
@@ -682,10 +682,26 @@ UserSchema.pre('remove', async function (next) {
         listIds.push(checkin.list);
       });
     });
+    let updates = {
+      count: -1
+    };
+    if (this.authOnly && !this.hidden) {
+      updates.countManager = -1;
+    }
+    if (!this.authOnly && !this.hidden) {
+      if (!this.is_orphan && !this.is_ghost) {
+        updates.countManager = -1;
+        updates.countVerified = -1;
+        updates.countUnverified = -1;
+      } else {
+        updates.countManager = -1;
+        updates.countVerified = -1;
+      }
+    }
```

Comment:
So is the idea that we're manually adjusting some values when performing operations that affect the counts? I ask for my own understanding.

Reply:
Yes correct. In this case, when a user is deleted from HID (or deletes his own account), the list counters of the lists the user is checked into are updated to reflect this.